RELEASE_NOTES.md

```diff
@@ -1,3 +1,30 @@
+## Cookstyle 6.14
+
+### 2 New Cops
+
+#### ChefCorrectness/MacosUserdefaultsInvalidType
+
+The `macos_userdefaults` resource prior to Chef Infra Client 16.3 would silently continue if invalid types were passed resulting in unexpected behavior. Valid values are `array`, `bool`, `dict`, `float`, `int`, and `string`. The `ChefCorrectness/MacosUserdefaultsInvalidType` will detect invalid types and autocorrect several common mistakes such using `integer` instead of `int`.
```

Comment:
```suggestion
The `macos_userdefaults` resource prior to Chef Infra Client 16.3 would silently continue if invalid types were passed resulting in unexpected behavior. Valid values are `array`, `bool`, `dict`, `float`, `int`, and `string`. The `ChefCorrectness/MacosUserdefaultsInvalidType` will detect invalid types and autocorrect several common mistakes such as using `integer` instead of `int`.
```